config/locales/en.yml

```diff
@@ -54,16 +54,56 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    claim_notification:
+      subject: "[%{app_name}] Works uploaded"
+      open_doors: "Open Doors"
+      open_doors_faq: "FAQ page"
+      contact_open_doors: "contact Open Doors"
+      ao3: "Archive of Our Own"
+      ao3_news: "AO3 News"
+      tutorials_page: "tutorials page"
+      support: "contact Support"
```

Comment:
It might make sense to call it `contact_support`, since we call the contact link for OD `contact_open_doors`, but it's not a big deal.